mappings/net/minecraft/server/integrated/IntegratedServerLoader.mapping

```diff
@@ -16,6 +16,9 @@ CLASS net/minecraft/unmapped/C_phqvxqun net/minecraft/server/integrated/Integrat
 	METHOD m_pemmxxxv askForBackup (Lnet/minecraft/unmapped/C_bpwagnit$C_unecclfw;ZLjava/lang/Runnable;Ljava/lang/Runnable;)V
 		ARG 2 legacyCustomized
 		ARG 3 callback
```

Comment:
```suggestion
		ARG 1 session
		ARG 2 legacyCustomized
		ARG 3 onProceeded
		ARG 4 onCancelled
```

Reply:
Are you wanting to add these mappings?

Reply:
Yeah, I did a `callback` search after suggesting changing that other param, and ended up mapping this.